sklearn/metrics/pairwise.py

```diff
@@ -187,17 +187,20 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
 
     Y : {array-like, sparse matrix}, shape (n_samples_2, n_features)
 
-    Y_norm_squared : array-like, shape (n_samples_2, ), optional
+    Y_norm_squared : array-like, shape (1, n_samples_2), optional
```

Comment:
I'm pretty sure this is tested to work with a 1-d array. Can we leave this shape alone?